test/remote/gateways/remote_payment_express_test.rb

```diff
@@ -138,6 +138,7 @@ def test_transcript_scrubbing
 
     assert_scrubbed(@credit_card.number, clean_transcript)
     assert_scrubbed(@credit_card.verification_value.to_s, clean_transcript)
+    assert_scrubbed(fixtures(:payment_express)[:password].to_s, clean_transcript)
```

Comment:
This can just be `assert_scrubbed(@gateway.options[:password]...`, which would make it consistent with other remote tests that do this (see e.g. authorizenet)